DOM - AutoIncrementFieldDescriptor describe IDFormat property (#2482)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.md

There are also special types of `FieldDescriptors` that are purpose-made to store a special value. These include:

> - **AutoIncrementFieldDescriptor**: Defines a field that will automatically get an incremented value assigned. When a `DomInstance` does not have a value for this field yet, it will get assigned a next save.

Reviewer: ```suggestion
- **AutoIncrementFieldDescriptor**: Defines a field that will automatically get an incremented value assigned. When a `DomInstance` does not have a value for this field yet, it will get assigned the next time the instance is updated.
```

>   - The `IDFormat` property is used to define a format the stored value. In this string "{0}" gets replaced the. If the `IDFormat` property is empty, then its value will not be formatted.

Reviewer: ```suggestion
  - The `IDFormat` property is used to define a string format for the integer value. In this string "{0}" gets replaced by that integer value. If the `IDFormat` property is empty, then its value will not be formatted.
```

    Some examples, assuming the next value is 10:

    | Description | Format | Result |
    |---|---|---|
    | Add a prefix and postfix | Pre-{0}-Post | Pre-10-Post |
    | Prefix with "REF-" and [format](https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-numeric-format-strings) the value | REF-{0:000000} | REF-000010 |
    | When no format is set the value is stored | | 10 |

>   - When the field is marked as soft-deleted, no values will get assigned when `DomInstances` get saved and its value will no longer increment.

Reviewer: ```suggestion
  - When the field is marked as soft-deleted, no values will get assigned when a `DomInstances` gets saved.
```
